src/modules/75-cf/pages/feature-flags/components/RenderFeatureFlag.tsx

```diff
@@ -0,0 +1,284 @@
+import React, { useEffect, useRef, useState, ReactElement } from 'react'
+import { Button, Container, Heading, Layout, Text, Utils, Popover } from '@harness/uicore'
+import { noop } from 'lodash-es'
+import { Classes, Position, Switch, PopoverInteractionKind } from '@blueprintjs/core'
+import { Color } from '@harness/design-system'
+import type { Cell } from 'react-table'
+import { useToaster } from '@common/exports'
+import type { Feature, GitSyncErrorResponse } from 'services/cf'
+import { useStrings, String } from 'framework/strings'
+import { ResourceType } from '@rbac/interfaces/ResourceType'
+import { PermissionIdentifier } from '@rbac/interfaces/PermissionIdentifier'
+import RBACTooltip from '@rbac/components/RBACTooltip/RBACTooltip'
+import { usePermission } from '@rbac/hooks/usePermission'
+import type { UseToggleFeatureFlag } from '@cf/hooks/useToggleFeatureFlag'
+import useActiveEnvironment from '@cf/hooks/useActiveEnvironment'
+import { featureFlagHasCustomRules, getErrorMessage, isFeatureFlagOn } from '@cf/utils/CFUtils'
+
+import SaveFlagToGitModal from '@cf/components/SaveFlagToGitModal/SaveFlagToGitModal'
+import { GIT_COMMIT_MESSAGES } from '@cf/constants/GitSyncConstants'
+import { GitDetails, GitSyncFormValues, GIT_SYNC_ERROR_CODE, UseGitSync } from '@cf/hooks/useGitSync'
+import type { UseGovernancePayload } from '@cf/hooks/useGovernance'
+import usePlanEnforcement from '@cf/hooks/usePlanEnforcement'
+import { FeatureIdentifier } from 'framework/featureStore/FeatureIdentifier'
+import { useFeature } from '@common/hooks/useFeatures'
+import { FeatureWarningTooltip } from '@common/components/FeatureWarning/FeatureWarningWithTooltip'
+import css from './RenderFeatureFlag.module.scss'
+
+export interface RenderFeatureFlagProps {
+  numberOfEnvs?: number
+  gitSync: UseGitSync
+  cell: Cell<Feature>
+  toggleFeatureFlag: UseToggleFeatureFlag
+  governance: UseGovernancePayload
+  refetchFlags: () => void
+}
+
+export const RenderFeatureFlag: React.FC<RenderFeatureFlagProps> = ({
```

Comment:
named export thingy again
```suggestion
export const RenderFeatureFlag: FC<RenderFeatureFlagProps> = ({
```